server/utils/validation.ts

```diff
@@ -92,11 +97,14 @@ export const errorMessage = (field: string, text: string): ErrorMessage => {
   }
 }
 
-const extractValidationErrors = (error: SanitisedError | Error) => {
+const extractValidationErrors = (error: SanitisedError | Error): Record<string, string> | string => {
   if ('data' in error) {
-    if (error.data['invalid-params']) {
+    if (error.data['invalid-params']?.length) {
```

Comment:
If we're checking the length here in order to see if it is an array I think it would be clearer to use `Array.isArray` which would also ensure it's not a string.